cmd/cat-main.go

```diff
@@ -243,7 +243,7 @@ func mainCat(ctx *cli.Context) error {
 	}
 
 	encKeyDB, err := parseEncryptionKeys(sseKeys)
-	fatalIf(err, "Unable to parse encryption keys")
+	fatalIf(err, "Unable to parse encryption keys:")
```

Comment:
You don't need to add ":" we do not follow that syntax. any particular reason for this change?

Reply:
looked more readable. can revert